app/src/pages/SingleContent.vue

```diff
@@ -22,9 +23,28 @@ const content = db.getBySlugAsRef<ContentDto>(props.slug);
 const tagsContent = ref<ContentDto[]>([]);
 
 watch(
-    content,
+    [content, appLanguageAsRef],
     async () => {
-        if (!content.value) return;
+        if (!content.value) {
+            return;
+        }
+
+        if (appLanguageAsRef.value?._id != content.value.language) {
```

Comment:
I don't think this should be in the same watcher as the one for `content`. If I am in an English article that doesn't have a Swahili translation, and change my language to Swahili, I get the notification. But when I reload the page, I get the notification twice, because this watcher triggers everytime either `content` or `appLanguageAsRef` changes. So I think at the very least this should be in it's own watcher.

Reply:
Although this issue is also something else: the notification will now always show, even if I'm not switching language. That could be the intended behaviour, just want to double check that we do this deliberately.